nncf/torch/graph/pattern_operations.py

```diff
@@ -11,68 +11,69 @@
  limitations under the License.
 """
 from nncf.common.graph.patterns import merge_two_types_of_operations
+from nncf.common.graph.patterns import GraphPattern
 
-LINEAR_OPERATIONS = {'type': ['linear',
```

Comment:
I think we should update patterns on Metatypes in this PR how it did in ONNX and OV

Reply:
The problem is that in contrast with ONNX backend, Torch backend uses operation names instead of metatypes for pattern matching. I suggest to cover this in a separate PR

Reply:
I know it, but the think that you need to do is only to change the name of the layer to Metatype.

Reply:
What do you mean by the new API has patterns only on Metatypes? Could you please show me this new API?

Reply:
I want to use torch patterns inside `MinMaxQuantization`, and for that I don't need to change types here. I would do this in a separate PR with pattern matching algorithm refactoring

Reply:
Please, look at this PR - https://github.com/openvinotoolkit/nncf/commit/54540ba1e1f50ad961c8cb3e2b285f361b332ac7#diff-62c24dee1521c104745fab8ceee4a346d952825ea04e70152033a1ceb78265afL16
This is how the migration of patterns was done for ONNX.

All matching logic already is implemented.  About what refactoring did you write?

Reply:
> This method, for example, could be simplified when all backends will be migrated on metatypes https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/graph/graph_matching.py#L76-L95

This is true, but it should be done only when ALL backends are migrated. If we add only Torch I don't see any refactoring in matching logic